### ISSUE TITLE
Remove use of types module constants in twisted/web

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8459]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8459|
|Type|enhancement|
|Created|2016-06-12 06:06:27Z|

The *types* module constants have been removed in Python 3.
For example using *types.TupleType* in Python 3 will give an *AttributeError*

These constants should be removed and replaced with the primitive type instead.  See:

http://www.diveintopython3.net/porting-code-to-python-3-with-2to3.html#types

<details><summary>Searchable metadata</summary>

```
trac-id__8459 8459
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__None None
time__1465711587104720 1465711587104720
changetime__1466479043641303 1466479043641303
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/179

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_2"></a>|@wsanchez set owner to @rodrigc|
|-|-|

Reviewed, good to merge.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc set status to `closed`|
|-|-|

In changeset 74b9622e1e00ac13a16d722af8cd1efaa524f4cd
```
#!CommitTicketReference repository="" revision="74b9622e1e00ac13a16d722af8cd1efaa524f4cd"
Merge branch '8459-rodrigc-types': Replace use of types constants with primitive types

Author: rodrigc
Reviewer: wsanchez
Fixes: #8459
```
